commands/ps/index.js

```diff
@@ -41,6 +29,41 @@ function printExtended (dynos) {
   })
 }
 
+function * printAccountQuota (heroku, app) {
+  if (app.process_tier !== 'free') {
+    return
+  }
+
+  let account = yield heroku.request({path: '/account'})
```

Comment:
@dickeyxxx I need the `account.id` from `/account` in order to form the path for this request, so I had to do them sequentially.

https://github.com/heroku/heroku-apps/blob/29eef6dc9905ace10e0a91756eb4548d4d1b84b1/commands/ps/index.js#L40 

That being said, I could bundle the account request in with these.

https://github.com/heroku/heroku-apps/blob/29eef6dc9905ace10e0a91756eb4548d4d1b84b1/commands/ps/index.js#L95

The reason I did not is because if the app is not free, then we can avoid the extra call to `/account` at the cost of doing another request batch.  Which do you think is better, avoiding the extra request to `/account` or doing it the same time as `/dynos` & `/app`?  I could go either way.

https://github.com/heroku/heroku-apps/blob/29eef6dc9905ace10e0a91756eb4548d4d1b84b1/commands/ps/index.js#L33